rollup.config.js

```diff
@@ -37,17 +46,32 @@ function strings({ include, exclude } = {}) {
   return {
     name: "strings",
 
+    /**
+     * @param {string} importee
+     */
     resolveId(importee) {
       if (importee.endsWith("!string")) {
+        // strip the `!string` from `importee`
+        importee = importee.slice(0, importee.lastIndexOf("!string"));
+        if (!importee.startsWith("gen/")) {
+          // this is a static asset which is located relative to the root of the source project
+          return path.resolve(path.join(process.env.BASEPATH, importee));
+        }
+        // ignoring the first part, which is "gen"
+        const [, ...importeeParts] = importee.split("/");
+        // generated assets will be output by TypeScript relative between the build path and the
```

Comment:
I think it's helpful to give examples
"relative between the build path (e.g. out/debug)"